package-lock.json

```diff
@@ -11838,13 +11845,29 @@
       }
     },
     "react-transition-group": {
-      "version": "4.1.0",
-      "resolved": "https://registry.npmjs.org/react-transition-group/-/react-transition-group-4.1.0.tgz",
-      "integrity": "sha512-/OITbogb3emGN49WaP7468QGSde7er5w6eIHldIDCSQBq/9QTSCzs8OgpgmOnaUXCXzBUcK1zoZ6DqRlM8CJtA==",
+      "version": "4.1.1",
+      "resolved": "https://registry.npmjs.org/react-transition-group/-/react-transition-group-4.1.1.tgz",
+      "integrity": "sha512-K/N1wqJ2GRP2yj3WBqEUYa0KV5fiaAWpUfU9SpHOHefeKvyrO+VrnMBML21M19QZoVbDZKmuQFHZYoMMi1xuJA==",
       "requires": {
+        "@babel/runtime": "^7.4.5",
         "dom-helpers": "^3.4.0",
         "loose-envify": "^1.4.0",
         "prop-types": "^15.6.2"
+      },
+      "dependencies": {
```

Comment:
This is new too. Not sure what it's about. Does this make sense to you?

Reply:
This is an instance where one of our deps decided to declare an implicit dependency explicitly, to deal with a bug, and was expected. See above.